src/olympia/devhub/api_views.py

```diff
@@ -0,0 +1,28 @@
+from olympia import amo
+from olympia.addons.views import AddonVersionViewSet
+from olympia.api.permissions import (
+    AllowAddonAuthor, AllowReviewer, AllowReviewerUnlisted, AnyOf)
+from olympia.devhub.models import ActivityLog
+from olympia.devhub.serializers import ActivityLogSerializer
+from olympia.versions.models import Version
+
+
+class VersionReviewNotesViewSet(AddonVersionViewSet):
+    permission_classes = [
+        AnyOf(AllowAddonAuthor, AllowReviewer, AllowReviewerUnlisted),
+    ]
+    serializer_class = ActivityLogSerializer
+    queryset = ActivityLog.objects.all()
+    filter = amo.LOG_REVIEW_QUEUE_DEVELOPER
+
+    def get_queryset(self):
+        self.get_addon_object()  # Just to force the permission checks.
+        if not hasattr(self, 'version_object'):
```

Comment:
That pattern was useful in `AddonVersionViewSet` because it was in a separate method that was re-used in a couple places, so we avoided doing the query twice. It's not useful here, at least not in the current state of the viewset.